pkg/get/get_test.go

```diff
@@ -1075,7 +1075,7 @@ func Test_DownloadDigitalOcean(t *testing.T) {
 
 	tool := getTool(name, tools)
 
-	const toolVersion = "1.46.0"
+	const toolVersion = "v1.46.0"
 	const urlTemplate = "https://github.com/digitalocean/doctl/releases/download/v1.46.0/doctl-1.46.0-%s-%s.%s"
```

Comment:
this should probably be moved away from urlTemplate since we're moving away from it in `tools.go`

The test table would then look like much like the other tests